README.md

```diff
@@ -4,16 +4,17 @@
 OAuth2 Provider for Finagle
 ---------------------------
 
-This is a [finagled](https://github.com/twitter/finagle) **OAuth2** _server-side_ provider based on the source code from 
-a [scala-oauth2-provider](https://github.com/nulab/scala-oauth2-provider) library. 
-Since [scala-oauth2-provider](https://github.com/nulab/scala-oauth2-provider) it involves: 
+This is a [finagled](https://github.com/twitter/finagle) **OAuth2** _server-side_ provider based on
+the source code from a [scala-oauth2-provider](https://github.com/nulab/scala-oauth2-provider)
+library. Since [scala-oauth2-provider](https://github.com/nulab/scala-oauth2-provider) it involves:
 
  - an asynchronous version of a `DataHandler` trait
  - asynchronous versions of both `TokenEndpoint` and `ProtectedResource` handlers
  - an asynchronous version of unit tests
  - four new finagled entities: `OAuth2`, `OAuth2Request`, `OAuth2Filter`, `OAuth2Endpoint`
 
-This makes the usage of this library very sleek from a finagled environment. The brief usage instruction looks as follows:
+This makes the usage of this library very sleek from a finagled environment. The brief usage
```

Comment:
finagled => Finagle

The brief usage... => Please see below for brief instructions: